modules/microshift-manifests-removal-scenario.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario_{context}"]
+== Manifest removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Use one of the following steps to create the `delete` subdirectories where the manifest is placed for deletion:
+* For RPM installation, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/etc/microshift/manifests/delete/`.
+* For OSTree deployments, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/usr/lib/microshift/manifests.d/delete/*` or `/usr/lib/microshift/manifests/delete/`.
```

Comment:
As mentioned in comment above - this is incorrect

Reply:
separate module created for OSTree